docs/releasenotes/0.8.0.rst

```diff
@@ -0,0 +1,34 @@
+0.8.0
+=====
+
+We are happy to announce another developmental release under the summer fellowship at `The Processing Foundation <https://processingfoundation.org/>`_.
+The project was supervised by `Mark Zhang <https://github.com/ziyaointl>`_. This release comes with an experimental 2D renderer using `Skia <https://skia.org/>`_.
+The skia 2D renderer shows significance improvements in both rendering and efficiency of the sketches.
+
+Bug Fixes
+--------------
+- #276
+- #311
+- #308
+- #309
+- #355
+
+New Features
+------------
+
+- Along with the snake_case convention we support the camelCase convention for most Processing APIs #280
```

Comment:
I think I don't remember a PR for this. Is this something you plan to add before the release?

Reply:
https://github.com/p5py/p5/pull/285

Reply:
Ahh I see. Thank you!